### DISCUSSION TITLE
QGis Crash importing Fiona

### DISCUSSION THREAD

**Discussion by Edcart** [Toblerity/Fiona#1214] [Q&A] [answered]
In Linux SO (ubuntu 22.04) My Qgis 28.3 crashes importing Fiona in Python shell. I installed Fiona 1.9.1 via "PIP"

**Comment by sgillies** [comment, marked as answer]
The Fiona packages on PyPI can't be used with QGIS because of conflicts between shared libraries that each rely on. QGIS provides Python bindings and it's better to use those: https://docs.qgis.org/3.22/en/docs/pyqgis_developer_cookbook/loadlayer.html#vector-layers.

If you really, really have to have the fiona package in QGIS, I think you will have to embrace the Conda package manager and get both QGIS and Fiona from [conda-forge](https://conda-forge.org/). Then they would mutually share library dependencies and you would avoid conflicts.
